dart/dynamics/Skeleton.hpp

```diff
@@ -159,8 +159,16 @@ class Skeleton :
   ConstSkeletonPtr getSkeleton() const;
 
   /// Get the mutex that protects the state of this Skeleton
+  /// \deprecated Use getStdMutex instead.
+  DART_DEPRECATED(6.4)
   std::mutex& getMutex() const;
 
+  /// Get the mutex that protects the state of this Skeleton
+  std::mutex& getStdMutex() const;
```

Comment:
If we change `getCustomMutex()` to `getMutexReference()`, then we won't need this function anymore.

Reply:
Removed.